Classes/Interfaces/Hook.php

```diff
@@ -0,0 +1,16 @@
+<?php
+declare(strict_types=1);
+
+namespace OliverKlee\Seminars\Interfaces;
```

Comment:
Please let's move this to the existing `OliverKlee\Seminars\Interfaces\Hook` sub-namespace.

Reply:
In master there is no such sub-namespace, yet. The existing one uses singular (`Interface`), which is a reserved word and thus unusable. I'm OK with using the `\Hook\` sub-namespace, but maybe `\Hooks\Interfaces\` would be a better place as the sub-namespace is new anyways?.

Reply:
You're right, there is no such namespace yet, and I was mistaken.

Yes, `Hooks\Interfaces\` sounds good.